react/src/components/molecules/ArrowNav/ArrowNav.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, text, select } from '@storybook/addon-knobs/react';
+import { action } from '@storybook/addon-actions';
+
+import ArrowNav from './index';
+import ArrowNavReadme from './ArrowNav.md';
+
+storiesOf('molecules', module)
+  .addDecorator(withKnobs)
+  .add('ArrowNav', withInfo({ ArrowNavReadme })(() => {
+    const props = {
+      href: text('Link href', ''),
```

Comment:
can you update these to match other knobs --> e.g. `ArrowNav.href`

Reply:
👍